.github/workflows/clippy_pr_changelog.yml

```
@@ -0,0 +1,56 @@
+name: Clippy changelog check
+
+on:
```

Let's also run this on 
```suggestion
on:
  merge_group:
```

It is a really cheap check. Just to be safe and because I don't know, if you can set a check as "Required" for enqueuing, but not in the merge queue.

---

Ok. I will have to see if the PR number is available in this case, it should be. 

(and I have to remove one unrelated change that went in in another file)